doc/jetty-config.md

```diff
@@ -39,6 +39,12 @@ that exceeds this value, Jetty will return an HTTP 413 Error response. This
 defaults to 8192 bytes, and only needs to be configured if an exceedingly large
 header is being sent in an HTTP Request.
 
+### `so-linger-seconds`
+
+This sets the TCP SO_LINGER time, in seconds, that the webserver uses for
+underlying socket connections.  Values less than 0 result in SO_LINGER
+being disabled.
```

Comment:
Yeah, I should definitely document what the default is - which is -1, a.k.a. "disabled".

I had left the description intentionally a little vague since I'm not totally confident that the behaviors I'm seeing at the TCP layer when I use this feature will be consistent across environments -- different OS distributions, etc.  I'll try to beef the description up a bit more, maybe with some caveats about how the actual behaviors may differ per environment.